src/app/(sidebar)/account/fund/page.tsx

```diff
@@ -80,6 +84,26 @@ export default function FundAccount() {
     return <SwitchNetwork />;
   }
 
+  const handleMuxedAccount = (muxedAddress: string) => {
+    if (muxedAddress.startsWith("M")) {
```

Comment:
can we use [isValidMed25519PublicKey](https://stellar.github.io/js-stellar-sdk/StrKey.html#.isValidMed25519PublicKey) method to check if it's a valid muxed address?

Reply:
ah i see. This is taken care of in `getPublicKeyError`. Thanks